client-ts/Microsoft.AspNetCore.SignalR.Client.TS/Microsoft.AspNetCore.SignalR.Client.TS.csproj

```diff
@@ -40,12 +40,17 @@
     <PropertyGroup>
       <ModuleVersion>@(ExecOutput)</ModuleVersion>
       <ModuleVersion Condition="'$(VersionSuffix)' != ''">$(ModuleVersion)-$(VersionSuffix)</ModuleVersion>
+      <IsGitRepository Condition="EXISTS('$(MSBuildThisFileDirectory)..\..\.git\HEAD')">true</IsGitRepository>
+      <IsGitRepository Condition="'$(IsGitRepository)'==''">false</IsGitRepository>
```

Comment:
nit: you could flip these and set unconditionally IsGitRepository to false and set it to true only if condition is met.